data/src/main/java/me/androidbox/data/local/dao/TaskDao.kt

```diff
@@ -0,0 +1,25 @@
+package me.androidbox.data.local.dao
+
+import androidx.room.*
+import kotlinx.coroutines.flow.Flow
+import me.androidbox.data.local.DatabaseConstant
+import me.androidbox.data.local.entity.TaskEntity
+
+@Dao
+interface TaskDao {
+    @Query("SELECT * FROM ${DatabaseConstant.TASK_TABLE}")
+    fun getTask(): Flow<List<TaskEntity>>
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertTask(taskEntity: TaskEntity)
+
+    /** TODO
+     * Delete only a single task
+     * */
+    @Delete
+    fun deleteTask(taskEntity: TaskEntity)
```

Comment:
Is this correct we only want to delete a single reminder

Reply:
Same as before